### PR TITLE
Add windows CI testing

### LINKED ISSUE #174: Add Windows CI testing
We are currently not testing (nor developing) `phys2bids` on Windows systems, leading to issues like e.g., #171. I think we should look into adding Windows CI testing using a service like [AppVeyor](https://www.appveyor.com/) or [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) (since TravisCI currently only has [minimal](https://docs.travis-ci.com/user/reference/windows/) Windows support).

## Context / Motivation
We shouldn't limit our users to those researchers working on Mac or Linux systems, so adding AppVeyor or Azure support / testing so that we can ensure our primary workflows are viable on Windows is critical.

## Possible Implementation
`nipy/nibabel` recently moved to Azure Pipelines from AppVeyor (relevant files: [1](https://github.com/nipy/nibabel/blob/master/.azure-pipelines/windows.yml) [2](https://github.com/nipy/nibabel/blob/master/azure-pipelines.yml)); we could look into simply copying the relevant portions of their framework for `phys2bids`.

### LINKED ISSUE #303: Run tests on Windows using CircleCI
## Detailed Description

Now that we have CircleCI running and is not giving us any problems, we should write some tests on Windows. This would make our testing suite only depend on CircleCI (i.e., we would not need Azure anymore). This way, we avoid having two different setups and two different places to check the logs.

## Possible Implementation

I would only run the integration tests on Windows to avoid having too many testing jobs on CircleCI. We currently have:

- 4 testing jobs, 
- the job to create the environment, 
- the "build the docs" job, 
- the style check job,
- the merge coverage job.

So, that's 8 jobs we currently have. Adding another 4 (2 integration + 2 unit test) jobs would be a bit redundant imo. However, I'd like to know what you guys think about this @tsalo @vinferrer @smoia .

The steps to implement this can be found on the [CircleCI documentation](https://circleci.com/docs/2.0/hello-world-windows/) page.

### PR DESCRIPTION
**Pull request by vinferrer**
Closes #303  

## Proposed Changes

  - Add Windows CI testing
  - one job for integration and another for unitest
  - delete azure pipelines when done

## Change Type

- [ ] `bugfix` (+0.0.1)
- [ ] `minor` (+0.1.0)
- [ ] `major`  (+1.0.0)
- [ ] `refactoring` (no version update)
- [x] `test` (no version update)
- [ ] `infrastructure` (no version update)
- [ ] `documentation` (no version update)
- [ ] `other`

## Checklist before review

- [x] I added everything I wanted to add to this PR.
- [ ] \[Code or tests only\] I wrote/updated the necessary docstrings.
- [x] \[Code or tests only\] I ran and passed tests locally.
- [ ] \[Documentation only\] I built the docs locally.
- [ ] My contribution is harmonious with the rest of the code: I'm not introducing repetitions.
- [ ] My code respects the adopted style, especially linting conventions.
- [x] The title of this PR is explanatory on its own, enough to be understood as part of a changelog.
- [x] I added or indicated the right labels.

- [ ] I added information regarding the timeline of completion for this PR.

- [ ] Please, comment on my PR while it's a draft and give me feedback on the development!

### CODE REVIEW

**Comment by vinferrer** [comment]
Wow, this was surprisingly easy, One integration test for windows already running in circle ci

File: .circleci/config.yml [hunk 1]
```diff
@@ -238,13 +239,58 @@ jobs:
...
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: Generate enviroment
+          command: |
+            python -m pip install --upgrade pip setuptools>=30.3.0 wheel
+            pip install -e ".[test]"
+      - run:
+          name: Run unittests
```
Comment (eurunuela) [inline .circleci/config.yml:278]:
```suggestion
          name: Run unit tests
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -238,13 +239,58 @@ jobs:
...
+          path: /tmp/data
+
+  windows_unittest_37:
+    build:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
```
Comment (eurunuela) [inline .circleci/config.yml:271]:
```suggestion
      - run: System info
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -238,13 +239,58 @@ jobs:
...
               - src/phys2bids
 
+  windows_integrationtest_37:
+    build:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
```
Comment (eurunuela) [inline .circleci/config.yml:249]:
```suggestion
      - run: System info
```

**Review by eurunuela — APPROVED** [review]
LGTM once the suggestions are added. This time your lowercase names are not escaping from me 😝

File: .circleci/config.yml (hunk 2 above)
Comment (eurunuela) [reply .circleci/config.yml]: Okay, not this one 😅

File: .circleci/config.yml (hunk 3 above)
Comment (eurunuela) [reply .circleci/config.yml]: Nor this one 😅

File: .circleci/config.yml (hunk 2 above)
Comment (vinferrer) [inline .circleci/config.yml:271]: this is a command, not a name

Comment (eurunuela) [reply .circleci/config.yml:271]: Yes, my bad!

**Comment by vinferrer** [comment]
@eurunuela , you wanted to recheck this

**Pull request merged**